source/_components/delijn.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "De Lijn"
+description: "Instructions on how to integrate De Lijn (Flemish public transport company) departure times into Home Assistant."
+date: 2019-05-23 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: "0.94"
+ha_category: 
+  - Transport
+ha_iot_class: Cloud Polling
+ha_qa_scale: 
+---
+
+The `delijn` sensor will give you the departure time of the next bus, tram or subway a specifi stop of the De Lijn public transport network in Flanders (Belgium).
+
+## {% linkable_title Setup %}
+
+Create a developer account at [De Lijn Open Data portal](https://data.delijn.be/) to get a free API subscription key.
+For valid stop IDs check for the 6 digits at the physical stops or visit the [stops page](https://www.delijn.be/en/haltes/) of the De Lijn website.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
```

Comment:
Only required items in the first example

Reply:
Thanks, adjusted in a new commit.